src/include/user/rtnr.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Realtek Semiconductor Corp. All rights reserved.
+ *
+ * Author: Ming Jen Tai <mingjen_tai@realtek.com>
+ */
+
+#ifndef __USER_RTNR_H__
+#define __USER_RTNR_H__
+
+#include <stdint.h>
+
+#define SOF_RTNR_MAX_SIZE 256
+
+struct sof_rtnr_params {
+	/* 1 to enable RTNR, 0 to disable it */
+	int32_t enabled;
+} __attribute__((packed));
```

Comment:
lets also align to 4 bytes.

Reply:
Could you please elaborate more? The size of sof_rtnr_params should be 4 bytes and the size of sof_rtnr_config should be 24 bytes.  Thanks.

Reply:
```__attribute__((packed, aligned(4)));```

Reply:
Got it, thanks.